src/main/java/org/medicmobile/webapp/mobile/GestureHandler.java

```diff
@@ -0,0 +1,24 @@
+package org.medicmobile.webapp.mobile;
+
+import android.view.MotionEvent;
+
+public class GestureHandler {
+
+	private float finger1AxisX;
+	private float finger2AxisX;
+	private static final float MIN_SWIPE_LENGTH = 200;
+
+	public GestureHandler(float finger1AxisX, float finger2AxisX) {
```

Comment:
Minor, but since the `isSwipeRight` method just takes a `MotionEvent` object, is there a reason we cannot just pass the same in here?